provider/consumer.py

```diff
@@ -65,20 +72,25 @@ def start(self):
     # should only be called by the Doctor thread
     def restart(self):
         if self.thread.desiredState() is Consumer.State.Dead:
-            logging.info('[{}] Request to restart a consumer that is already slated for deletion.'.format(self.trigger))
+            logging.info("[{}] Request to restart a consumer "
+                         "that is already slated for "
+                         "deletion.".format(self.trigger))
             return
 
         with self.__lock:
             self.__restartCount += 1
 
-        logging.info('[{}] Quietly shutting down consumer for restart'.format(self.trigger))
+        logging.info("[{}] Quietly shutting down consumer for "
+                     "restart".format(self.trigger))
         self.thread.setDesiredState(Consumer.State.Restart)
         self.thread.join()
-        logging.info('Consumer has shut down')
+        logging.info("Consumer has shut down")
```

Comment:
hmmm... quote usage throughout these changes is inconsistent. What is the reason for changing some but not others?